src/Runner.Worker/ExecutionContext.cs

```diff
@@ -72,6 +72,9 @@ public interface IExecutionContext : IRunnerService
 
         IExecutionContext FinalizeContext { get; set; }
 
+        // Only Composite Level ExecutionContext has CompositeSteps (public setter for handler)
+        List<IStep> CompositeSteps { get; set; }
```

Comment:
Any reason we have to store `CompositeSteps` as part of `ExecutionContext`?
The reason I store the JobStep is for register `post` steps.
what about `CompositeStepsRunner. RunAsync(ExecituonContext context, IStep compositeSteps)`, since you know what's going to run up front (for now. 😄 ).